docs/tutorials/13_trotterQRTE.ipynb

```diff
@@ -0,0 +1,983 @@
+   "metadata": {},
+   "source": [
+    "Knowing this, we can have a look at the circuit that performs a single Trotter step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "ee8f1ba9-3c5d-41fe-9f93-1fe735216dd9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Trotter step with Lie-Trotter\n",
+      "-----------------------------\n",
+      "\n",
+      "                  Depth: 7\n",
+      "             Gate count: 17\n",
+      "    Nonlocal gate count: 5\n",
+      "         Gate breakdown: RZ: 6, RX: 6, RZZ: 5\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA3UAAAGwCAYAAAAOtEwOAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8pXeV/AAAACXBIWXMAAA9hAAAPYQGoP6dpAABjuUlEQVR4nO3de1hVZf7//9fmLCKelZOKiIwnDuGhQK0om8Yp0UytRi0Ksyyzg6mVU0yf+Y0ZmilmaTNmefXJKHUcOziVqd+ozGRSIzMUlXArpAgYqMhp//7gE8UAysbN3qzN83FdXDOs+77Xeq+9VrpfrrXuZbJYLBYBAAAAAAzJxdEFAAAAAACajlAHAAAAAAZGqAMAAAAAAyPUAQAAAICBEeoAAAAAwMAIdQAAAABgYIQ6AAAAADAwQh0AAAAAGBihDgAAAAAMjFAHAAAAAAZGqAMAAAAAAyPUAQAAAICBEeoAAAAAwMAIdQAAAABgYIQ6AAAAADAwQh0AAAAAGBihDgAAAAAMjFAHAAAAAAZGqAMAAAAAAyPUAQAAAICBEeoAAAAAwMAIdQAAAABgYIQ6AAAAADAwQh0AAAAAGBihDgAAAAAMjFAHAAAAAAZGqAMAAAAAAyPUAQAAAICBEeoAAAAAwMAIdQAAAABgYIQ6AAAAADAwQh0AAAAAGBihDgAAAAAMjFAHAAAAAAZGqAMAAAAAAyPUAQAAAICBEeoAAAAAwMAIdQAAAABgYG6OLgBAy2KxSFXljq6i6VzcJZPJ0VUYnxHPA449AKC1ItQBqKWqXNqe4ugqmi5uluTq4egqjM+I5wHHHgDQWnH7JQAAAAAYGKEOAAAAAAyMUAcAAAAABkaoAwAAAAADI9QBAAAAgIER6gAAAADAwHilAQCb2Xd4hx5fGVdrmZdHWwV1DdOo6KkaN/whubryx44z4xwAAMD++JsVgM3FRd2hYf3+KIssKizO0yf/WauV7z2mnJMH9OiEVx1dHuyAcwAAAPsh1AGwub6B0Ro1eErN72NiH1Bicj9t+fofuvsPf1MHn64OrA72wDkAAID98EwdgGbXxqOt+vW6ShaLRSdOH3Z0OXAAzgEAAJoPoQ6AXeT+3xd5X+9ODq4EjsI5AABA8+D2SwA2V1p+TmfO5stiqX6e6r2dK5V1fI/69RimoK5hji4PdsA5AACA/bSKUJefn6/k5GRt3LhRZrNZXbt21fjx47VgwQLNmjVLr732mpYvX66ZM2c6ulTAKaz9OElrP06qtWzEoPF66JYVDqoI9sY5AACA/Th9qNu7d69Gjx6tvLw8tW3bVgMGDNCJEyeUkpKiw4cPq6CgQJIUFRXl2EKdWNk56cR30qksqaJUcnGX2nWVgqIkXz9HV4fmcNOV03V1xERVVJXraG6GUnc8r/wzZnm4e9X0yTiSpqdWj64ztqKyTFVVlfooudKeJcPGGnMOlFVc0ANLoxV3xZ80+fr5NcuT305QUclPWjBtiyNKBwDAcJw61OXn52vMmDHKy8vT7NmzlZSUpHbt2kmSkpOTNW/ePLm5uclkMikiIsLB1TofS5WU9Zl0bI9U9V/fz4t/qg567QOkQTdJbdo7pkY0j8AufRUdNkqSNKzfaA3qPUKPvjxCyzbcr/lT3pYkhYeM1Ht/K6k1Lv/MCT2YMkRjY7lqbnSNOQc83Dw19/a1mv3y1bqq/83qExCpL77bpK8OvKdXH8twZPkAABiKU0+UMmvWLJnNZs2cOVOLFy+uCXSSNHfuXEVGRqqiokLBwcHy9fV1YKXOx2KRvvtQ+jG9bqD7rTMnpN1vSeeK7FYaHGBgcKxGRU/Vjn2p2p/9Zb19yiou6Nm14zUoeIT+dP1Tdq4Qza2hcyAsaLAmXPO4kt++U6eKzFq6froeumWFurQPcGC1AAAYi9OGugMHDig1NVVdunTRc889V2+fwYMHS5IiIyNrLT969Kji4+PVrl07dezYUXfeeadOnz7d7DU7kx+/ln76oXF9y85K+zZWX9mD85o86mm5uLjqjY+eqbd92Yb7VVZeqjm3vW7fwmA3DZ0Dk0f9Wa4ubpqx9ApFhsYpLup2B1UIAIAxOW2oW7dunaqqqjR58mT5+PjU26dNmzaSaoe64uJixcXFyWw2a926dXr11VeVlpamm2++WVVVpI7GqKqUcr6xbszZAun00eapBy1DYJdQxUXerj1ZnyrjSFqttn9+nqJdB97Xswmb5OXh7aAK0dwaOgfcXN01IDhWZ87m68YhdzuwQgA
```

Comment:
```suggestion
    "There exists a second-order formula, called the Suzuki-Trotter decomposition [3], and can be used in Qiskit by means of the [SuzukiTrotter](https://qiskit.org/documentation/stubs/qiskit.synthesis.SuzukiTrotter.html) class. Using this formula, a second order decomposition is:\n",
```
I had marked this in the review above but it came out odd - hopefully this shows the change correctly now